packages/rich-text-editor/src/RichTextEditor/keymap.ts

```diff
@@ -0,0 +1,78 @@
+import { EditorState, Transaction } from "prosemirror-state"
```

Comment:
I've changed my mind and decided that this keymap file belongs in Kaizen instead of the toolkit. My reasoning is that it's quite tied to things that we currently have in Kaizen, like the name of marks in the schema. It's also consistent with what I've done with the inputrules file here.

Perhaps we could revisit this in the future and see if there's something we could extract out here to the toolkit so make for consistent keymaps across custom RTEs.

Reply:
I think that's fair. As you said we can extract it later when we've built out the functionality of the out of the box solution - I think by then we'll have a firm understanding of what needs to be and can be passed into the more composable solution